src/transformers/models/biogpt/modeling_biogpt.py

```diff
@@ -544,7 +544,11 @@ def forward(
             inputs_embeds = self.embed_tokens(input) * self.embed_scale
 
         if attention_mask is None:
-            attention_mask = torch.ones(inputs_embeds.shape[:2], dtype=torch.bool, device=inputs_embeds.device)
```

Comment:
This was a bug previously. The attention_mask should be equal to input_embeds + past_key_values length.